packages/regenesis-surgery/scripts/handlers.ts

```diff
@@ -101,7 +184,98 @@ export const handlers: {
   [AccountType.UNVERIFIED]: () => {
     return undefined // delete the account
   },
-  [AccountType.VERIFIED]: () => {
+  [AccountType.VERIFIED]: (account, data) => {
+    if (account.storage) {
+      for (const pool of data.pools) {
+        // Check for references to modified values in storage.
+        for (const [key, val] of Object.entries(account.storage)) {
+          // TODO: Do we need to do anything if these statements trigger?
```

Comment:
i THOUGHT these references are what we address in the next loop - why are these problematic here?